spec/api/courses_spec.rb

```diff
@@ -46,50 +40,13 @@
                 example: { page: 2, per_page: 10 },
                 description: "Pagination options to navigate through the collection."
 
-      response "200", "The collection of courses." do
+      response "200", "The collection of courses for the specified year." do
```

Comment:
```suggestion
      response "200", "The collection of courses." do
```
based on 
https://api2.qa.publish-teacher-training-courses.service.gov.uk/api-reference.html#recruitment_cycles-year-providers-get